### DISCUSSION TITLE
Pygbag index.html not working

### DISCUSSION THREAD

**Discussion by brrpy** [pygame-web/pygbag#151] [Help zone]
I'm trying to run a pygbag test project but it gives the alert here: Error occurred: undefined is not an object (evaluating 'i.fs[t]') when I run it on Safari. When I run it on Chrome, it gives the alert Error occurred: Failed to fetch. My directory system:
main folder
- main.py
- assets
   - image

without the build folder. Am I doing something wrong? My OS: MacOS 11.7.10. I usually run the command
`python3 -m pygbag drillgame` to package the project. I have implemented asyncio into the file as well, and I am running pygame-ce 2.4.0.
I also tried using the debug menu, which seems to be fine. It does give the warning `no fc_cache font cache file at {fc_cache}`. However, it runs fine on localhost:8000.

**Comment by pmp-p** [comment]
> However, it runs fine on localhost:8000

Yes it is made for that and nothing more. 

If you want to run it elsewhere you need web hosting with SSL certificate  and upload index.html + apk file.

 You can find tutorials on https://pygame-web.github.io for itch.io and github pages.

running from just html file will only happens in final version see
https://github.com/pygame-web/pygbag/issues/77
